docs/en/latest/plugins/openid-connect.md

```diff
@@ -61,12 +61,12 @@ description: OpenID Connect allows the client to obtain user information from th
-| proxy_opts.proxy_opts.https_proxy    | string   | False    |                       | http://proxy-server:port         | HTTPS proxy server address.                                                                                                                                                                                                           |
-| proxy_opts.http_proxy_authorization  | string   | False    |                       | Basic [base64 username:password] | Default `Proxy-Authorization` header value to be used with `http_proxy`.                                                                                                                                                              |
-| proxy_opts.https_proxy_authorization | string   | False    |                       | Basic [base64 username:password] | As `http_proxy_authorization` but for use with `https_proxy` (since with HTTPS the authorisation is done when connecting, this one cannot be overridden by passing the `Proxy-Authorization` request header).                         |
+| unauth_action                        | string   | False    | "auth"                |  ["auth","deny","pass"]            | Specify the response type on unauthenticated requests. "auth" redirects to identity provider, "deny" results in a 401 response, "pass" will allow the request without authentication.                                                 |
+| proxy_opts                           | object   | False    |                       |                                  | HTTP proxy that the OpenID provider is behind.                                                                                                                                                                                  |
+| proxy_opts.http_proxy     | string   | False    |                       | http://proxy-server:port         | HTTP proxy server address.                                                                                                                                                                                                            |
+| proxy_opts.https_proxy    | string   | False    |                       | http://proxy-server:port         | HTTPS proxy server address.                                                                                                                                                                                                           |
+| proxy_opts.http_proxy_authorization  | string   | False    |                       | Basic [base64 username:password] | Default `Proxy-Authorization` header value to be used with `http_proxy`. Can be overriden with custom `Proxy-Authorization` request header.                                                                                                                                                              |
```

Comment:
References for `http_proxy_authorization` and `https_proxy_authorization`:

For `http_proxy_authorization` and `https_proxy_authorization`:

* https://github.com/ledgetech/lua-resty-http/blob/master/t/16-http-proxy.t#L413-L513 
* https://github.com/ledgetech/lua-resty-http/blob/master/README.md?plain=1#L209-L215